Tests/TuistGeneratorTests/ProjectMappers/ResourcesProjectMapperTests.swift

```diff
@@ -85,6 +85,10 @@ final class ResourcesProjectMapperTests: TuistUnitTestCase {
             "SKIP_INSTALL": "YES",
             "CODE_SIGNING_ALLOWED": "NO",
         ])
+
+        XCTAssertEqual(resourcesTarget.infoPlist, .extendingDefault(with: [
```

Comment:
Don't really like this test. Should we, probably have a full acceptance / integration one, when we check what will be in the final product?
If so, were should I put it?